app/app.go

```diff
@@ -245,7 +245,7 @@ func NewApp(c Config, db *sql.DB) (*App, error) {
 
 // WaitForStartup will wait until the startup sequence is completed or the context is expired.
 func (a *App) WaitForStartup(ctx context.Context) error {
-	return a.mgr.WaitForStartup(log.WithLogger(ctx, a.cfg.Logger))
+	return a.mgr.WaitForStartup(a.Context(ctx))
```

Comment:
I'm getting `app/app.go:248:32: a.Context undefined (type *App has no field or method Context)` when attempting to start the app, so I think this line should be reverted?

Reply:
```suggestion
	return a.mgr.WaitForStartup(log.WithLogger(ctx, a.cfg.Logger))
```

Reply:
Oops yeah I noticed that and meant to add it, nice catch 👍 thanks